src/js/components/Grid/StyledGrid.js

```diff
@@ -216,6 +216,7 @@ const areasStyle = (props) => {
 const StyledGrid = styled.div
   .withConfig(styledComponentsConfig)
   .attrs((props) => ({
+    ...enhancePropsWithTheme(props),
```

Comment:
why is this one formatted different just trying to understand

Reply:
Because `attrs` was already used for passing `aria-label` (see the line below). So using `attrs(enhancePropsWithTheme)` would result in losing that property.

Reply:
oh gotcha thank you for clarifying